options/options.js

```diff
@@ -1,12 +1,14 @@
 const OPTION_DISABLE_KEYBOARD_SHORTCUTS = "disableKeyboardShortcuts";
-const CSS_ID_DISABLE_KEYBOARD_SHORTCUTS = "disableKeyboardShortcuts";
+const OPTION_ADD_BACKGROUND_TABS_TO_FRONT = "addBackgroundTabsToFront";
```

Comment:
I used two constants instead of one to avoid confusion, please follow the same pattern.